pkg/controllers/deprovisioning/controller.go

```diff
@@ -153,11 +153,21 @@ func (c *Controller) executeCommand(ctx context.Context, d Deprovisioner, comman
 	}
 
 	for _, oldNode := range command.nodesToRemove {
+		owningProvisioner := "N/A"
```

Comment:
Why do we need "N/A"? Why not just leave it as the empty string?

Reply:
We could use an empty string, I've used it mostly because of https://github.com/aws/karpenter-core/blob/main/pkg/controllers/metrics/state/scraper/node.go#L223

Reply:
Got it, we should probably change that there as well. I'm more in favor of empty string if it doesn't eixst